puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/vars/foreman-installer-develop-release.groovy

```diff
@@ -4,3 +4,5 @@ def project_name = 'foreman-installer'
 def obal_package_name = 'foreman-installer'
 def build_rpm = true
 def build_deb = true
+def source_type = 'rake'
```

Comment:
At some point I'd like to optimize here. In Foreman we can also use `rake -f Rakefile.dist pkg:generate_source` and that doesn't need a bundle install which makes it much faster. I'm thinking of introducing a `bundle-rake` source type vs `rake` but this is a future optimization.